Diagnostic/Utils/XmlUtil.py

```diff
@@ -40,3 +42,10 @@ def addElement(xml,path,el,selector=[],addOnlyOnce=False):
 def createElement(schema):
     return ET.fromstring(schema)
 
+
+def removeElement(tree, parent_path, removed_element_name):
+    parents = tree.findall(parent_path)
+    for parent in parents:
+        removed_elements = parent.findall(removed_element_name)
+        for element in removed_elements:
+            parent.remove(element)
```

Comment:
Is this the right pattern? Does removing an element reshape the tree such that previously-found elements cannot simply be handed to parent.remove()? Or do you need something that looks more like this:
```
for parent in parents:
    element = parent.find(removed_element_name)
    while element is not null:
        parent.remove(element)
        element = parent.find(removed_element_name)
```